File: 7-kyu/check-three-and-two/python/solution.py
def check_three_and_two(array):
    array_set = list(set(array))
    return len(array_set) == 2 and ((array.count(array_set[0]) == 2 and array.count(array_set[1]) == 3) or \
        (array.count(array_set[1]) == 2 and array.count(array_set[0]) == 3))

File: 7-kyu/check-three-and-two/javascript/solution.js
const checkThreeAndTwo = array => {
  const count = (a, val) => a.filter(elem => elem === val).length;
  const aCount = count(array, "a");
  const bCount = count(array, "b");
  const cCount = count(array, "c");
  const checkThree = aCount === 3 || bCount === 3 || cCount === 3;
  const checkTwo = aCount === 2 || bCount === 2 || cCount === 2;
  return checkThree && checkTwo;
};
